nifi-commons/nifi-property-utils/src/main/java/org/apache/nifi/properties/AbstractBootstrapPropertiesLoader.java

```diff
@@ -121,8 +136,7 @@ private File getBootstrapFile(final String bootstrapPath) throws IOException {
             if (confDir.exists() && confDir.canRead()) {
                 expectedBootstrapFile = new File(confDir, BOOTSTRAP_CONF);
             } else {
-                logger.error("Cannot read from bootstrap.conf file at {} -- conf/ directory is missing or permissions are incorrect", confDir.getAbsolutePath());
-                throw new IOException("Cannot read from bootstrap.conf");
+                throw new IOException(String.format("Cannot read {} directory for {}", confDir, bootstrapPath));
```

Comment:
It looks like the placeholder values should be changed:
```suggestion
                throw new IOException(String.format("Cannot read %s directory for %s", confDir, bootstrapPath));
```